news/2021-05-07-monthly-beatmapping-contest-may-2021.md

```diff
@@ -0,0 +1,112 @@
+
+### <a id="limitation"></a>Limitation
+
+We have two goals here at MBC headquarters: to create mapping contests every month and to ensure those contests aren't repetitive. It's a tough act to balance, which is why we have **limitations** — mapping restrictions that submissions need to follow, like "straight sliders only" or "circles cannot be used" or "you must map with your eyes closed". We unfortunately haven't tried that last one yet.
+
+Below are the limitations for this month!
+
+**[osu! MBC](LINK) limitations:**
+
+- Total slider velocity (SV) cannot be higher than **1.4**
+
+What's "total slider velocity"? It's not an official term, but we're talking about a map's **base SV** (set in the Timing tab of the editor) combined with any **SV multipliers** (green lines set in the F6 timing panel).
+
+So for example:
+
+- If base SV is **1.4**, SV multipliers can't be higher than **1x**. (1.4 x 1 = 1.4 "total SV")
+- If base SV is **2.8**, your multipliers can't be higher than **0.5x** (2.8 x 0.5 = 1.4 "total SV").
+
+Pretty straightforward, right? For those who'd rather not think, the default base SV is 1.4, so don't touch that and only use SV multipliers below 1x. You'll be fine.
+
+**[osu!taiko MBC](LINK) limitation:**
+
+- All patterns on snaps faster than 1/2 must have an **even** number of notes.
+
+![](/wiki/shared/news/2021-05-07-monthly-beatmapping-contest-may-2021/taiko-examples.png)
+
+If you're brave enough to map anything other than a swing song with this limitation, we'll give them an honorary 0.01 MBC points for the attempt.
+
+**[osu!mania MBC](LINK) limitation:**
+
+- Long notes cannot be used.
+
+### <a id="songs"></a>Songs
+
+Each MBC provides beatmappers a choice between songs from one of osu!'s [Featured Artists](https://osu.ppy.sh/beatmaps/artists).
+
+This month's artist is guitar savant [**Masahiro "Godspeed" Aoki**](https://osu.ppy.sh/beatmaps/artists/121). These three songs are available for mapping in the osu!, osu!taiko, and osu!mania contests:
+
+#### Frostbite (188 BPM)
+
+Beatmap template download: [Short version](LINK) (1:47) / [Full version](https://assets.ppy.sh/artists/136/Veiled/Masahiro%20Godspeed%20Aoki%20-%20Frostbite.mp3) (3:49)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/136/Veiled/Masahiro%20Godspeed%20Aoki%20-%20Frostbite.mp3">
+</audio>
+
+#### The Game Changer (280 BPM)
+
+Beatmap template download: [Short version](LINK) (2:24) / [Full version](https://assets.ppy.sh/artists/136/Songs/Masahiro%20Godspeed%20Aoki%20-%20The%20Game%20Changer.mp3) (3:44)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/136/Songs/Masahiro%20Godspeed%20Aoki%20-%20The%20Game%20Changer.mp3" type="audio/mpeg">
+</audio>
+
+#### busy come, busy go (instrumental version) (270 BPM)
+
+Beatmap template download: [Short version](https://assets.ppy.sh/artists/136/The%20Game%20Changer/Masahiro%20Godspeed%20Aoki%20-%20busy%20come,%20busy%20go%20(instrumental%20version).osz) (2:15)
```

Comment:
busy come, busy go's length is 1:52 iirc (not 2:15)

![image](https://user-images.githubusercontent.com/36564236/117407168-5ee49480-af38-11eb-9ddf-a725f1c046b0.png)